apps/style/common.scss

```diff
@@ -670,6 +673,15 @@ input[type="radio"] {
   text-align: justify;
   -ms-text-justify: distribute-all-lines;
   text-justify: distribute-all-lines;
+
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+
+  @supports (gap: 0 9px) {
+    gap: 0 9px;
+  }
```

Comment:
Can we remove the `@supports` feature query here (and keep the one up at line 265)? Browser that do not support `gap` will ignore the property anyway.

Reply:
Oh yes, of course! Done :)